app/api/schema/orders.py

```diff
@@ -47,8 +47,8 @@ def initial_values(self, data):
             data['payment_mode'] = 'free'
         return data
 
-    id = fields.Str(dump_only=True)
-    identifier = fields.Str(dump_only=True)
+    id = fields.Str()
+    identifier = fields.Str()
```

Comment:
Why remove dump_only from here? Don't change things unless you realize they are absolutely necessary